cmd/genji/shell/command.go

```diff
@@ -52,43 +52,54 @@ func runTablesCmd(db *genji.DB, cmd []string) error {
 
 // displayTableIndex prints all indexes that the given table contains.
 func displayTableIndex(db *genji.DB, tableName string) error {
-	err := db.View(func(tx *genji.Tx) error {
-		t, err := tx.GetTable(tableName)
+	ctx := context.Background()
+
+	q := fmt.Sprintf("SELECT * FROM __genji_indexes WHERE table_name = '%s'", tableName)
```

Comment:
For your culture, you should **never** do this in a user facing application. This allows anyone to run [sql injections](https://www.w3schools.com/sql/sql_injection.asp)

Here for example, let's imagine that this function is called by a command or a web app and that `tableName` is passed by a user. Someone could write the following:

```
'; DROP TABLE users; SELECT '1
```

This would generate this:

```sql
SELECT * FROM __genji_indexes WHERE table_name = ''; DROP TABLE users; SELECT '1'
```

This is a valid sql query, which would destroy the users table.

To avoid that, always use query parameters

`res, err := db.Query(ctx, "SELECT * FROM __genji_indexes WHERE table_name = ?", tableName)`

Reply:
Thank you Asdine.